lib/linguist/memory_blob.rb

```diff
@@ -0,0 +1,73 @@
+require 'linguist/blob_helper'
+
+module Linguist
+  # A MemoryBlob is a wrapper around the content of a file to make it quack
+  # like a Grit::Blob. It provides the basic interface: `name`,
+  # `data`, `path` and `size`.
+  class MemoryBlob
```

Comment:
Yep, that could make sense since `Linguist::FileBlob` inherits it.
I'll change it.